<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>prefDialog</class>
 <widget class="QDialog" name="prefDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preference</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>11</number>
     </property>
     <property name="topMargin">
      <number>11</number>
     </property>
     <property name="rightMargin">
      <number>11</number>
     </property>
     <property name="bottomMargin">
      <number>11</number>
     </property>
     <item>
      <widget class="QLabel" name="TextLabel1_2">
       <property name="text">
        <string>Paste Word-Wrap At</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="wordLineEdit3"/>
     </item>
     <item>
      <widget class="QCheckBox" name="smartCheckBox">
       <property name="text">
        <string>Enable Smart Word-Wrap</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" rowspan="2">
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="wheelCheckBox">
       <property name="text">
        <string>Alternative Mouse Wheel Control</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="urlCheckBox">
       <property name="text">
        <string>Detect URL When Mouse Move</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="saveCheckBox">
       <property name="text">
        <string>Save Message To File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="blinkCheckBox">
       <property name="text">
        <string>Blink Tab When Receive Message</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="warnCheckBox">
       <property name="text">
        <string>Warn When Exit Connected</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="aacheckBox">
       <property name="text">
        <string>AntiAlias</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="trayCheckBox">
       <property name="text">
        <string>Show As Tray When Close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="clearCheckBox">
       <property name="text">
        <string>Clear ZModem And HTTP Directory When Exit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="enqCheckBox">
       <property name="text">
        <string>Reply when ENQ received</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="asciiEnhanceCheckBox">
       <property name="text">
        <string>ASCII Art Enhance Mode</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1" rowspan="2" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Sound When Receive Messages</string>
       </property>
       <layout class="QGridLayout">
        <property name="leftMargin">
         <number>11</number>
        </property>
        <property name="topMargin">
         <number>11</number>
        </property>
        <property name="rightMargin">
         <number>11</number>
        </property>
        <property name="bottomMargin">
         <number>11</number>
        </property>
        <property name="spacing">
         <number>6</number>
        </property>
        <item row="0" column="0">
         <widget class="QRadioButton" name="noneRadioButton">
          <property name="text">
           <string>None</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="beepRadioButton">
          <property name="text">
           <string>Beep</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QRadioButton" name="fileRadioButton">
          <property name="text">
           <string>Sound File</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLineEdit" name="wavefileLineEdit"/>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="selectsoundPushButton">
          <property name="text">
           <string>Configure</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>11</number>
       </property>
       <property name="topMargin">
        <number>11</number>
       </property>
       <property name="rightMargin">
        <number>11</number>
       </property>
       <property name="bottomMargin">
        <number>11</number>
       </property>
       <item>
        <widget class="QLabel" name="TextLabel1">
         <property name="text">
          <string>Input Method</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ximComboBox">
         <item>
          <property name="text">
           <string>GBK</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>BIG5</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Display Offset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="displayOffsetSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>5</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="verticalDisplayGroupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="title">
        <string>Vertical Display</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QRadioButton" name="topRadioButton">
          <property name="text">
           <string>Top</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="middleRadioButton">
          <property name="text">
           <string>Middle</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QRadioButton" name="bottomRadioButton">
          <property name="text">
           <string>Bottom</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>287</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="textLabel1">
         <property name="text">
          <string>Zmodem Directory</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel9">
         <property name="text">
          <string>StyleSheet File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="TextLabel3">
         <property name="text">
          <string>HTTP Browser</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel1_4">
         <property name="text">
          <string>Editor &amp; Args</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QLineEdit" name="zmodemLineEdit"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLineEdit" name="styleSheetLineEdit"/>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLineEdit" name="httpLineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLineEdit" name="editorLineEdit"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="editorArgLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="browsePushButton">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="styleSheetPushButton">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="choosehttpPushButton">
         <property name="text">
          <string>Choose</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="editorPushButton">
         <property name="text">
          <string>Choose</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="Line" name="Line1">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>11</number>
     </property>
     <property name="topMargin">
      <number>11</number>
     </property>
     <property name="rightMargin">
      <number>11</number>
     </property>
     <property name="bottomMargin">
      <number>11</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okPushButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>wordLineEdit3</tabstop>
  <tabstop>smartCheckBox</tabstop>
  <tabstop>wheelCheckBox</tabstop>
  <tabstop>urlCheckBox</tabstop>
  <tabstop>saveCheckBox</tabstop>
  <tabstop>blinkCheckBox</tabstop>
  <tabstop>warnCheckBox</tabstop>
  <tabstop>aacheckBox</tabstop>
  <tabstop>trayCheckBox</tabstop>
  <tabstop>clearCheckBox</tabstop>
  <tabstop>noneRadioButton</tabstop>
  <tabstop>beepRadioButton</tabstop>
  <tabstop>fileRadioButton</tabstop>
  <tabstop>selectsoundPushButton</tabstop>
  <tabstop>wavefileLineEdit</tabstop>
  <tabstop>ximComboBox</tabstop>
  <tabstop>zmodemLineEdit</tabstop>
  <tabstop>browsePushButton</tabstop>
  <tabstop>httpLineEdit</tabstop>
  <tabstop>choosehttpPushButton</tabstop>
  <tabstop>okPushButton</tabstop>
  <tabstop>cancelPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>noneRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>wavefileLineEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>noneRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>selectsoundPushButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>beepRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>wavefileLineEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>beepRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>selectsoundPushButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>413</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>wavefileLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>467</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>selectsoundPushButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>502</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>493</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
